src/catalog/src/memory/objects.rs

```diff
@@ -536,7 +536,7 @@ pub struct Table {
     pub create_sql: Option<String>,
     pub desc: RelationDesc,
     #[serde(skip)]
-    pub defaults: Vec<Expr<Aug>>,
+    pub defaults: Option<Vec<Expr<Aug>>>,
```

Comment:
Feels like this field should move in the `TableDataSource::TableWrites` enum, since it is only valid when that enum has that variant

Reply:
Yeah good point that is much cleaner, will refactor